api/v1/status_routes.go

```diff
@@ -41,6 +44,19 @@ func HandleGetStatus(rw http.ResponseWriter, r *http.Request, p httprouter.Param
 	_, _ = rw.Write(data)
 }
 
+func getFirstExternallyControlledExecutor(
+	execScheduler lib.ExecutionScheduler,
+) (*executor.ExternallyControlled, error) {
+
+	executors := execScheduler.GetExecutors()
+	for _, s := range executors {
+		if mex, ok := s.(*executor.ExternallyControlled); ok {
+			return mex, nil
+		}
+	}
+	return nil, fmt.Errorf("a externally-controlled executor needs to be configured for live configuration updates")
```

Comment:
Maybe use `errors.New`? The error message does not have any formatting here.

Reply:
Resolved by https://github.com/loadimpact/k6/pull/1007/commits/119fc185e6fc93521b60819d032613018a9d6dd6